sympy/integrals/prde.py

```diff
@@ -320,36 +320,62 @@ def prde_no_cancel_b_small(b, Q, n, DE):
         A = A.row_join(zeros(A.rows, m)).col_join(c.row_join(-c))
         return (H, A)
     else:
-        # TODO: implement this (requires recursive param_rischDE() call)
-        raise NotImplementedError
-
+        Q0 = [q.eval(0).as_poly(DE.t) for q in Q]
+        Q0 = [tuple([i.as_poly(DE.t) for i in q.as_numer_denom()]) for q in Q0]
+        ba, bd = b.as_expr().as_numer_denom()
+        ba, bd = ba.as_poly(DE.t), bd.as_poly(DE.t)
+        F, B = param_rischDE(ba, bd, Q0, DE)
+        if all(qi.is_zero for qi in Q):
+            dc = -1 if all((derivation(f, DE) + b*f).is_zero for f in F) else 0
+        else:
+            dc = max([qi.degree(DE.t) for qi in Q])
+        M = zeros(dc + 1, m + len(F) + 1)
+        M_ = Matrix(dc + 1, m, lambda i, j: Q[j].nth(i))
+        M[0, 0] = M_
```

Comment:
This line gives me an error with `prde_no_cancel_b_small(Poly(0, t), [Poly(2*t, t)], 2, DE)` with `DE = DifferentialExtension(extension={'D': [Poly(1, t)]})`.